src/models/apps/expense-split-app/expense.model.js

```diff
@@ -0,0 +1,67 @@
+import mongoose, { Schema } from "mongoose";
+import {
+  AvailableExpenseTypes,
+  AvailablePaymentMethods,
+  ExpenseTypes,
+  PaymentMethods,
+} from "../../../constants.js";
+
+const expenseSchema = mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+    },
+    groupId: {
+      type: Schema.Types.ObjectId,
+      ref: "ExpenseGroup",
+    },
+    Amount: {
```

Comment:
Why we have capitalized fields? keep the consistency. We are keeping camel casing for field names in the models.